contracts/test/ValidatorWalletTest.t.sol

```diff
@@ -0,0 +1,164 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity ^0.8.17;
+
+import "forge-std/Test.sol";
+import "../src/rollup/ValidatorWallet.sol";
+import "../src/rollup/ValidatorWalletCreator.sol";
+import "./RollupMock.sol";
+
+contract ValidatorWalletTest is Test {
+    // Contract instances
+    ValidatorWalletCreator creator;
+    ValidatorWallet wallet;
+    RollupMock rollupMock1;
+    RollupMock rollupMock2;
+
+    // Set initial executor and owner
+    address public executor = 0x5B38Da6a701c568545dCfcB03FcB875f56beddC4;
+    address public owner = 0xAb8483F64d9C6d1EcF9b849Ae677dD3315835cb2;
```

Comment:
These are some very specific addresses lol usually we can just use sth like addr(1337); no big deal tho